### PR TITLE
many fixes

### PR DESCRIPTION
**Pull request by anewton1998**
some final textual fixes and changes before conferring with the AD.

### CODE REVIEW

File: rpp-charter.md [hunk 1]
```diff
@@ -3,46 +3,50 @@
...
 The RPP working group is focused on designing a new protocol intended to co-exist alongside EPP, 
 supporting diverse needs in the ecosystem. As such, considerations for replacing EPP or migration 
-scenarios are outside the scope of this work.
+scenarios away from EPP are outside the scope of this work.
 
 The REGEXT working group is chartered to maintain and standardize extensions to EPP. Consequently, 
 any extensions or changes to EPP, including those related to RPP functionality that do not exist in EPP, 
-are explicitly out of scope for the RPP working group.
+are explicitly out of scope for the RPP working group. Some IETF participants active in the REGEXT
+working group will be involved in RPP working groups, and coordination between the two
```
Comment (mwullink) [inline rpp-charter.md:48]: "will be involved in RPP working groups" -> "will be involved in the RPP working group"

File: rpp-charter.md [hunk 2]
```diff
@@ -3,46 +3,50 @@
...
-to be known collectively as RESTful Provisioning Protocol (RPP).
+to be known collectively as the RESTful Provisioning Protocol (RPP).
 These specifications will specifically target a REST architecture high on the Richardson Maturity Model
 using HTTPS and JSON. 
 
-The working group will evaluate for possible inclusion of extensibility considerations, allowing
-future use cases beyond the domain name provisioning as well as adding
-or extending operations and data models defined in the core set of protocol specifications.
+In addition to use cases and scenarios for domain name registration, this working group will
+consider features and extension mechanisms useful for other types of Internet registration areas
```
Comment (mwullink) [inline rpp-charter.md:27]: "Internet registration areas" -> "registration areas" 

maybe future use cases may also include registration for non-Internet related resources?

File: rpp-charter.md [hunk 3]
```diff
@@ -77,13 +83,11 @@ of the chairs with the responsible Area Director:
 * core architecture
 * specifications necessary to implement the core architecture
 * specifications regarding the provisioning of domain names using RPP
-* mappings between RPP and EPP.
+* mappings between RPP and EPP
 
-The working group may consider work for the provisioning of other types of Internet registries (e.g. IP addresses)
-with consultation from the responsible Area Director.
+The working group may consider work for the provisioning of other types of Internet registries (e.g. IP addresses
```
Comment (mwullink) [inline rpp-charter.md:88]: again, same comment, is RPP limited to Internet resources or can it be used for anything requiring a central registry function?

File: rpp-charter.md (hunk 2 above)
Comment (pawel-kow) [reply rpp-charter.md:27]: I think "Internet" is what stays in concern of IETF.
I am more concerned about "will evaluate for possible inclusion of extensibility" changed to "will
consider features and extension mechanisms". The previous one was for me more of obligation to evaluate, now is sounds like obligation to consider them. My preference would be to stay with the previous one or to change from will to may, especially that deliverables read "The working group may consider work for the provisioning of other types of Internet registries".

Comment (anewton1998) [reply rpp-charter.md:27]: > "Internet registration areas" -> "registration areas"
> 
> maybe future use cases may also include registration for non-Internet related resources?

This is to purposefully limit the scope to the things the IETF concerns itself with. Any charter that is too broad is likely to be rejected.



> I think "Internet" is what stays in concern of IETF. I am more concerned about "will evaluate for possible inclusion of extensibility" changed to "will consider features and extension mechanisms". The previous one was for me more of obligation to evaluate, now is sounds like obligation to consider them. My preference would be to stay with the previous one or to change from will to may, especially that deliverables read "The working group may consider work for the provisioning of other types of Internet registries".

"Will evaluate" does not obligate the working group to produce anything, and neither does the new wording. But both do say the working group will do something. The original wording was clunky IMHO.

**Comment by rpp-ietf** [comment]
Andy,

I believe that "will evaluate for possible inclusion of extensibility" and its subsequent obligation as a requirement for the working group.  I don’t support softening this language.  We have EPP that has benefited from the pre-defined set of features and forms of extensibility and we’re learning the lesson now with the lack of pre-defined forms of extensibility in RDAP.  We need to evaluate the extensibility from the start.

Thanks,

--

JG

***@***.***

James Gould
Fellow Engineer
***@***.******@***.***>

703-948-3271
12061 Bluemont Way
Reston, VA 20190

Verisign.com<http://verisigninc.com/>

From: Andrew Newton ***@***.***>
Reply-To: SIDN/ietf-wg-rpp-charter ***@***.***>
Date: Monday, December 2, 2024 at 2:55 PM
To: SIDN/ietf-wg-rpp-charter ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] [rpp] Re: [SIDN/ietf-wg-rpp-charter] many fixes (PR #45)


Caution: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.


@anewton1998 commented on this pull request.

________________________________

In rpp-charter.md<https://secure-web.cisco.com/1y8bpRW_taPGzYZPnLAXkIMKk98EC_5byOZX_XCverkQedTsHSTMH1La_V9swIKbOChukdQGak3ETREgL-abxojMnCK4thZLMDJlhVWDK9U_p5sPv3Yoidj2wBEouzELzT8ZjaE-2bmXiSarqeCjMZSLQixCXMQteRPUkJdr_8W3-3UifSwXdxRSrblgnfM9NNRKrlEoHe7OavmN1eGByQHWT0xcwLkfE_PILMb5AiWE4WBXVeWPbkmmVTRi5GNr0lw6XSs8TDwKLqJB1w6YU0Rpg5U4ga5d_KfVyyogo798/https%3A%2F%2Fgithub.com%2FSIDN%2Fietf-wg-rpp-charter%2Fpull%2F45%23discussion_r1866524109>:

>  These specifications will specifically target a REST architecture high on the Richardson Maturity Model

 using HTTPS and JSON.



-The working group will evaluate for possible inclusion of extensibility considerations, allowing

-future use cases beyond the domain name provisioning as well as adding

-or extending operations and data models defined in the core set of protocol specifications.

+In addition to use cases and scenarios for domain name registration, this working group will

+consider features and extension mechanisms useful for other types of Internet registration areas

"Internet registration areas" -> "registration areas"

maybe future use cases may also include registration for non-Internet related resources?

This is to purposefully limit the scope to the things the IETF concerns itself with. Any charter that is too broad is likely to be rejected.

I think "Internet" is what stays in concern of IETF. I am more concerned about "will evaluate for possible inclusion of extensibility" changed to "will consider features and extension mechanisms". The previous one was for me more of obligation to evaluate, now is sounds like obligation to consider them. My preference would be to stay with the previous one or to change from will to may, especially that deliverables read "The working group may consider work for the provisioning of other types of Internet registries".

"Will evaluate" does not obligate the working group to produce anything, and neither does the new wording. But both do say the working group will do something. The original wording was clunky IMHO.

**Pull request merged**